cpp/source/deal.II/mp_values.cc

```diff
@@ -11,12 +11,21 @@ namespace cap
 {
 
 template class MPValuesParameters<2>;
-template class MPValues<2>;
 template class MPValuesParameters<3>;
-template class MPValues<3>;
+template class SuperCapacitorMPValues<2>;
+template class SuperCapacitorMPValues<3>;
+template class InhomogeneousSuperCapacitorMPValues<2>;
+template class InhomogeneousSuperCapacitorMPValues<3>;
 template class PorousElectrodeMPValues<2>;
 template class PorousElectrodeMPValues<3>;
 template class MetalFoilMPValues<2>;
 template class MetalFoilMPValues<3>;
 
+template class CompositeMat<2>;
+template class CompositeMat<3>;
+template class CompositePro<2>;
+template class CompositePro<3>;
+template class UniformConstantMPValues<2>;
```

Comment:
what does uniformconstant mean? are the value constant everywhere, i.e., it's homogeneous or is it a uniform distribution? If it is a uniform distribution, use uniformdistribution.